src/containers/SnippetArea.jsx

```diff
@@ -1,12 +1,13 @@
-import React, { PropTypes } from 'react';
-import { connect } from 'react-redux';
 import axios from 'axios';
-import _ from 'lodash'
+import _ from 'lodash';
 import { CardText, Snackbar } from 'material-ui';
-import { browserHistory } from 'react-router'
+import React, { PropTypes } from 'react';
```

Comment:
I think React should be the first import here. Makes it obvious that this is React code.